packages/frameworks-sveltekit/src/lib/client.ts

```diff
@@ -31,6 +32,40 @@ export type SignInAuthorizationParams =
   | Record<string, string>
   | URLSearchParams
 
+/**
+ * Fetch webauthn options from server and prompt user for authentication or registration.
+ * Returns either the completed WebAuthn response or an error request.
+ *
+ * @param providerId provider ID
+ * @param options SignInOptions
+ * @returns WebAuthn response or error
+ */
+async function webAuthnOptions(providerId: string, options?: SignInOptions) {
+  const { startAuthentication, startRegistration } = await import(
+    "@simplewebauthn/browser"
+  )
+  const baseUrl = `${base}/auth/`
```

Comment:
What if I set a different `basePath` in the `SvelteKitAuthConfig`?

Reply:
Good question.. was giving this some thought and the `SvelteKitAuthConfig` and sveltekit set `base` aren't necessarily in sync.

So I was thinking about implementing something similar to what we have in `next-auth` with the `apiBaseUrl` helper that parses the `__NEXTAUTH` config object throughout. What do you think?